src/apps/dictionaries/pages/EditDictionaryPage.tsx

```diff
@@ -96,6 +97,8 @@ const EditDictionaryPage: React.FC<Props> = ({
     linkedSource,
     createAndAddLinkedSource
   ]);
+  
+  useEffect(() => resetEditDictionary, []);
```

Comment:
`resetEditDictionary` also needs to be added to `ActionProps` and the start of the `EditDictionaryPage` function.